web/controllers/page_controller.ex

```diff
@@ -26,7 +30,9 @@ defmodule Askbywho.PageController do
       {:ok, email} -> # Inserted or updated with success
         redirect(conn, to: page_path(conn, :share, email.id))
       {:error, changeset} -> # Something went wrong
-        render(conn, "index.html", changeset: changeset, action: page_path(conn, :create))
+        brands = Repo.all(Brand)
```

Comment:
This chunk of code seems to be repeated a bit.  Should consider refactoring.  Also, any reason why the "create" method renders the list?  Is it so that the javascript can load all the values?